jubeatfesto.html

```diff
@@ -8,28 +8,82 @@
         <script type="text/javascript">
             window.addEventListener("load", function() {
                 new PatchContainer([
-                    new Patcher("jubeat.dll", "2022-05-24", [
+                    new Patcher("jubeat.dll", "2022-05-24, 2022-02-16, 2021-10-19", [
                         {
-                            name: "Skip Tutorial",
-                            patches: [{ offset: 0xD746F, off: [0x0F, 0x85], on: [0x90, 0xE9] }],
+                          name: "Disable Tutorial",
+                          patches: [{offset: 0xD746F, off: [0x0F, 0x85, 0x7F, 0x01, 0x00, 0x00], on: [0xE9, 0x80, 0x01, 0x00, 0x00, 0x90]}]
```

Comment:
I notice this hex edit is actually different, do you know if it works any differently? everything else is OK

Reply:
it works same